rfc/new-artifacts-service/README.md

```diff
@@ -0,0 +1,207 @@
+# Separate artifacts to new service
+
+| | |
+|-|-|
+| **Status**     | **Proposed**, Accepted, Implemented, Obsolete |
+| **RFC #**      | _Update after PR has been made_ |
+| **Author(s)**  | [Nemesis Osorio](https://github.com/nemesisOsorio), [Sergio Quintero](https://github.com/sergio-quintero) |
+| **SIG / WG**   | _Applicable SIG(s) or Working Group(s)_ |
+
+## Overview
+
+Users with large context pipelines do not have a good experience with pipeline executions. <br/>
+Context and outputs in pipeline executions can contain large artifacts/objects such as k8s manifest (yaml/json files). <br/>
+Spinnaker passes these artifacts/objects in request and response bodies across some services. <br/>
+The idea is to create a new service to store these artifacts/objects
+and only pass the `id`s of those artifacts/objects in the request/response bodies and context/outputs.
+
+Deck users do not click to see the produced artifact/object in every single pipeline execution.
+With this approach, artifacts will be loaded on demand (lazy loading).
```

Comment:
One thought, should this be "selectable" by artifact type?  I think not, but had to toss this out there - is there a state where we NEED artifact data LIKE Git information for the ui to know how it's triggered.

Also would this impact "Trigger" artifacts?

Reply:
I do not think we need a filter by artifact type.
For trigger artifacts, we've added a question to the `Known Unknowns` section, I'll keep digging in the code and hopefully add an answer.

Reply:
I added an answer for trigger artifacts